etc/c/curl.d

```diff
@@ -879,8 +881,10 @@ enum CurlOption {
   /** The CApath or CAfile used to validate the peer certificate
      this option is used only if SSL_VERIFYPEER is true */
   cainfo = 10065,
-  /** 66 = OBSOLETE */
-  /** 67 = OBSOLETE */
+  /** $(UL */
```

Comment:
Oops, right. Fixing.

BTW, are those even supposed to be doc comments? Doesn't seem like it, but I don't really know this module very well.